lib/ansible/modules/group.py

```diff
@@ -35,6 +35,12 @@
         type: str
         choices: [ absent, present ]
         default: present
+    force:
+        description:
+            - Whether to delete a group even if it is the primary group of a user.
+            - Only applicable on platforms which implement a --force flag on the group deletion command.
+        type: bool
+        default: false
```

Comment:
There should be `version_added: 2.14` for the new option.

Reply:
Added